packages/stepper/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@crave/farmblocks-stepper",
+  "version": "1.0.0",
+  "description": "A React Component for displaying step to step actions",
+  "author": "Crave Food Systems and AUTHORS",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "files": ["AUTHORS", "LICENSE", "README.md", "CHANGELOG.md", "lib"],
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "test": "jest",
+    "clean": "rm -rf lib",
+    "build": "yarn clean && babel src -d lib --ignore '*.test.js,*.story.js'",
+    "prestorybook": "cp -Rf ../../.storybook .",
+    "storybook": "start-storybook -p 6006"
```

Comment:
this is minor, as we rarely launch storybook from inside a package subfolder, but we should pass the assets I guess, replace this line with:

```
   "storybook": "start-storybook -p 6006 -s .storybook/assets",
```